src/test/java/io/xpire/testutil/TypicalItems.java

```diff
@@ -56,12 +73,15 @@ private TypicalItems() {} // prevents instantiation
     public static Xpire getTypicalExpiryDateTracker() {
         Xpire edt = new Xpire();
         for (Item item : getTypicalItems()) {
-            edt.addItem(item);
+            Item copyItem = new Item(item);
+            edt.addItem(copyItem);
         }
         return edt;
     }
 
     public static List<Item> getTypicalItems() {
-        return new ArrayList<>(Arrays.asList(EXPIRED_MILK, BANANA, APPLE, EXPIRED_ORANGE, EXPIRING_FISH));
+        return new ArrayList<>(Arrays.asList(EXPIRED_MILK, BANANA, APPLE, EXPIRED_ORANGE, EXPIRING_FISH, DUCK,
```

Comment:
you may want to look out for failed test cases for the add, delete and tag commands as we made use of the index of the typical items list. since the list is arranged by alphabetical order, renaming these variables may change the item index of the test cases that we have done >-<

Reply:
yes I will fix that!